audience_manager/form-tracker.rst

```diff
@@ -1,101 +1,150 @@
+:term:`attributes<attribute>`.
+
+.. note::
+    Creates or updates :term:`user` :term:`custom attribute` for each field in the form. :term:`Attribute` name is taken
+    from ``<input>`` or ``<textarea>`` tag ``name`` attribute. Form elements without ``name`` attribute are not tracked.
 
 Supported browsers
 ------------------
-All modern browsers (Chrome, Firefox, Safari, Edge). Internet Explorer from version 8 and above.
+All modern browsers: Chrome, Firefox, Safari, Edge. Internet Explorer from version 8 and above.
+
+.. todo:: Move information about supported browsers to document about PPMS in general since it's common to whole system.
 
 Private information
 -------------------
-Form tracker is trying to automatically detect fields containing private information and exclude them. Following fields
-are always removed from tracked data:
+Form Tracker is trying to automatically detect fields containing :term:`user` private information and ignores them.
+Following data is never send to Audience Manager:
 
-* "password" or "hidden" inputs
-* Credit card number
-* Credit card validation code
+.. todo:: Check what is legal term for ignored information's.
 
-Loading snippet
----------------
-Add following snippet on a page to start using Audience Manager Form Tracker.
+- Value from input with ``password`` or ``hidden`` type.
+- Credit card number (heuristic detection).
+- Credit card validation code (heuristic detection).
+- Data from ignored fields (see :ref:`AM-FT-optional-configuration`).
 
-The code should be added near the top of the <head> tag and before any other script or CSS tags. PPMS configuration
-additionally requires 2 changes in example code.
+.. note::
 
-Website ID
-``````````
-Replace 'XXX-XXX-XXX-XXX-XXX' with your Piwik Website ID. Example::
+    Heuristic detection makes best effort to automatically detect and ignore mentioned field types, but it doesn't
+    guarantee success. Additionally payment forms usually contain more fields with private information (e.g. address)
+    so it's recommended to ignore such forms using :ref:`AM-FT-optional-configuration`.
 
-    "efcd98a5-335b-48b0-ab17-bf43f1c542be"
+Installation
+------------
+This section describes how to install Form Tracker client code on your page.
 
-PPMS domain
-```````````
-Replace 'ppms.example.com' with your PPMS tracker domain. Please note that this value is used in 3 places:
+Tag Manager installation
+````````````````````````
+This is recommended way to install Form Tracker using PPMS stack. Tag Manager allows access to all Form Tracker options
+using friendly UI and will make sure that it's configuration is up to date.
 
-* 'https://ppms.example.com/audiences/static/widget/audience-manager.form.min.js'
-* 'https://ppms.example.com/audiences/tracker/'
-* 'https://ppms.example.com/audiences/static/widget/'
+.. todo:: Get access to TM on testing.piwik.pro and write instructions.
```

Comment:
@RomanSek you don't need to write instruction. Just link the article on Help Center: https://help.piwik.pro/audience-manager/capturing-data-forms/